add BEI doc (#405)
basetenlabs/truss-examples: 11-embeddings-reranker-classification-tensorrt/README.md

# BEI with Baseten

> This is a collection of BEI deployments with Baseten. BEI is Baseten's solution for production-grade deployments via TensorRT-LLM.

Reviewer: ```suggestion
This is a collection of BEI (Baseten Embeddings Inference) model implementations for deployment to Baseten. BEI is Baseten's solution for production-grade embeddings inference using TensorRT-LLM.
```

With BEI you get the following benefits:

> - *lowest-latency inference* across any embedding solution (vLLM, SGlang, Infinity, TEI, Ollama)*1

Reviewer: ```suggestion
- *Lowest-latency inference* across any embedding solution (vLLM, SGlang, Infinity, TEI, Ollama)<sup>1</sup>
```

> - *highest-throughput inference* across any embedding solution (vLLM, SGlang, Infinity, TEI, Ollama) - thanks to XQA kernels, FP8 and dynamic batching.*2

Reviewer: ```suggestion
- *Highest-throughput inference* across any embedding solution (vLLM, SGlang, Infinity, TEI, Ollama) - thanks to XQA kernels, FP8 and dynamic batching.<sup>2</sup>
```

> - high parallelism: up to 1400 client embeddings per second

Reviewer: ```suggestion
- High parallelism: up to 1400 client embeddings per second
```

> - cached model weights for fast vertical scaling and high availability - no Hugging Face hub dependency at runtime

Reviewer: ```suggestion
- Cached model weights for fast vertical scaling and high availability - no Hugging Face hub dependency at runtime
```

## Reranker Deployments:
 - [BAAI/bge-reranker-large](https://github.com/basetenlabs/truss-examples/tree/main/11-embeddings-reranker-classification-tensorrt/BEI-baai-bge-reranker-large)
 - [cross-encoder/ms-marco-MiniLM-L-6-v2-reranker](https://github.com/basetenlabs/truss-examples/tree/main/11-embeddings-reranker-classification-tensorrt/BEI-cross-encoder-ms-marco-minilm-l-6-v2-reranker)

## Text Sequence Classification Deployments:
 - [ProsusAI/finbert-classification](https://github.com/basetenlabs/truss-examples/tree/main/11-embeddings-reranker-classification-tensorrt/BEI-prosusai-finbert-classification)
 - [SamLowe/roberta-base-go_emotions-classification](https://github.com/basetenlabs/truss-examples/tree/main/11-embeddings-reranker-classification-tensorrt/BEI-samlowe-roberta-base-go_emotions-classification)
 - [Skywork/Skywork-Reward-Llama-3.1-8B-v0.2-Reward-Model](https://github.com/basetenlabs/truss-examples/tree/main/11-embeddings-reranker-classification-tensorrt/BEI-skywork-skywork-reward-llama-3.1-8b-v0.2-reward-model)

> * measured on H100-HBM3 (bert-large-335M, for MistralModel-7B: 9ms)

Reviewer: ```suggestion
<sup>1</sup> measured on H100-HBM3 (bert-large-335M, for MistralModel-7B: 9ms)
```

> ** measured on H100-HBM3 (leading model architecture on MTEB, MistralModel-7B)

Reviewer: ```suggestion
<sup>2</sup> measured on H100-HBM3 (leading model architecture on MTEB, MistralModel-7B)
```

Author: markdown god
